generator/api/src/test/java/io/fabric8/maven/generator/api/support/AbstractPortsExtractorTest.java

```diff
@@ -1,23 +1,22 @@
 package io.fabric8.maven.generator.api.support;
```

Comment:
You forgot to add license header in this file ;)

Reply:
I'm not an initial author of this file, but I'll add the license header.

Reply:
oops